modules/gapi/include/opencv2/gapi/gcompiled.hpp

```diff
@@ -208,6 +208,14 @@ class GAPI_EXPORTS GCompiled
     // FIXME: Why it requires compile args?
     void reshape(const GMetaArgs& inMetas, const GCompileArgs& args);
 
+    /**
+     * @brief Prepare inner kernels states for a new stream.
```

Comment:
It will be great if this comments also explains what the `stream` is.

Reply:
Please check if new explanation is enough?

Reply:
Fixed with your help, thanks a lot! Just added small corrections to your text, please check if you agree with them